activerecord/lib/active_record/connection_adapters/abstract/schema_creation.rb

```diff
@@ -78,8 +78,8 @@ def quote_table_name(name)
             @conn.quote_table_name name
           end
 
-          def type_to_sql(type, limit, precision, scale)
-            @conn.type_to_sql type.to_sym, limit, precision, scale
+          def type_to_sql(*args)
```

Comment:
An extra array param is passed below. But that should just be added to this definition instead of splatting the arguments.